packages/components/src/components/hds/form/super-select/multiple/field.ts

```diff
@@ -6,14 +6,30 @@
 import Component from '@glimmer/component';
 import { ID_PREFIX } from '../../label/index.ts';
 
+import type { ComponentLike } from '@glint/template';
+import type { HdsFormErrorSignature } from '../../error/index.ts';
 import type { HdsFormFieldSignature } from '../../field/index.ts';
+import type { HdsFormHelperTextSignature } from '../../helper-text/index.ts';
+import type { HdsFormLabelSignature } from '../../label';
 import type { HdsFormSuperSelectMultipleBaseSignature } from './base.ts';
+import type { Select as PowerSelect } from 'ember-power-select/components/power-select';
+import type { HdsYieldSignature } from '../../../yield/index.ts';
 
 interface HdsFormSuperSelectMultipleFieldSignature {
   Args: HdsFormSuperSelectMultipleBaseSignature['Args'] &
     HdsFormFieldSignature['Args'];
   Blocks: {
-    default: [unknown];
+    default: [
+      {
+        Label?: ComponentLike<HdsFormLabelSignature>;
+        HelperText?: ComponentLike<HdsFormHelperTextSignature>;
+        Error?: ComponentLike<HdsFormErrorSignature>;
+        Options?: ComponentLike<HdsYieldSignature>;
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        options?: any;
```

Comment:
Could the options be unknown?

Reply:
It could, but I'm not convinced we want to enforce type-checking on our consumers. To me, it's a more explicit way of saying that they can use any format for the `options` object. If you feel strongly about it, I'm happy to go with `unknown` – I know it's preferred, but not convinced in this scenario.

Reply:
🤷🏽 I think it is a decision we should make as a team. are most people using typescript? if so, it doesn't seem unreasonable to make them type check the options.

Reply:
Agree, let's bring it to the eng sync. However, I'm afraid I need a 'yes' sooner to unblock HCP, so I f9241750ca60b76f08ede92725e5be6d8cf81ef7 in the hope of an approval

Reply:
Agree, let's bring it to the eng sync. However, I'm afraid I need a 'yes' sooner to unblock HCP, so I f9241750ca60b76f08ede92725e5be6d8cf81ef7 in the hope of an approval